storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -24,15 +24,7 @@ To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage
 
```

Comment:
need to update the OCP version and the CSI driver version above (3.0.2)
cc @gnufied

Reply:
Done!